contracts/src/ExampleRollup.sol

```diff
@@ -19,8 +18,9 @@ contract ExampleRollup {
 
```

Comment:
on Line 15, the InvalidProof error type isn't used anywhere

Reply:
Thanks, will remove this!

Reply:
https://github.com/EspressoSystems/sequencer-example-l2/pull/71/commits/c62b0d6687fed93fda761a2a59398db444612945